### DISCUSSION TITLE
[Projects Beta] Make it easier to add issues to projects

### DISCUSSION THREAD

**Discussion by markcarroll** [community/community#6833] [Projects and Issues] [answered]
Adding issues to a project is painful. It always has been.  Hopefully this new version will fix this but so far it is not apparent. 

Ideally we can set the workflow such that all issues added to a repository can be automatically added to a project.  
Next would be adding filters so that all new issues of a certain criteria are added.  
Then add the ability to add all features in a milestone, or the milestone itself to the project. 
Lastly, adding a master issue should also add all the child issues for tracking.

**Comment by alexcnichols** [comment, marked as answer]
👋🏼 Improvements for auto adding issues to projects (beta), and more, are coming soon! Check out this roadmap item for more and subscribe to be notified if/when things change. https://github.com/github/roadmap/issues/286 :octocat: 

Marking this as a duplicate, in favor of this other discussion:

- https://github.com/github/feedback/discussions/5378

**Reply by markcarroll** [reply]
Appreciate the link, however I am not sure all the ideas I listed above have been fully captured. Specifically:

- Add all issues in a milestone to a project
- Add all child issues from a parent issue to a project
- Add option to bulk add issues to a project from the issues list screen (like you can assign, add labels etc)

**Reply by ant-04** [reply]
> 👋🏼 Improvements for auto adding issues to projects (beta), and more, are coming soon! Check out this roadmap item for more and subscribe to be notified if/when things change. [github/roadmap#286](https://github.com/github/roadmap/issues/286) :octocat:
> 
> Marking this as a duplicate, in favor of this other discussion:
> 
> * [Automatically add items from repositories (projects beta) #5378](https://github.com/github/feedback/discussions/5378)

+1 on setting a project as default when creating issues
![image](https://user-images.githubusercontent.com/65191196/148483181-1f9552ff-ff26-47e4-b619-f197f99d9232.png)

**Reply by aeneasr** [reply]
Please make it possible to add issues from other orgs. Sometimes we rely on issues which are not in the org of the project. This is possible through the UI, but not through GraphQL at the moment. It is a very useful feature if your product management goes beyond a single organization.

See also: https://github.com/github/feedback/discussions/9764

**Reply by danielhoherd** [reply]
We're coming up on 1 year since https://github.com/github/roadmap/issues/286 was created and there has been zero movement after day 1.
